doc/Scorer.rst

```diff
@@ -1,64 +1,83 @@
 .. _scorer-scripts:
 
-External scorer scripts
-=======================
+How to Train a Language Model
+=============================
 
-🐸STT pre-trained models include an external scorer. This document explains how to reproduce our external scorer, as well as adapt the scripts to create your own.
+Introduction
+------------
 
-The scorer is composed of two sub-components, a KenLM language model and a trie data structure containing all words in the vocabulary. In order to create the scorer package, first we must create a KenLM language model (using ``data/lm/generate_lm.py``, and then use ``generate_scorer_package`` to create the final package file including the trie data structure.
+This document explains how to train and format a language model for deployment.
```

Comment:
"format" a language model?

Reply:
OK I see the usage below. I think "package" is a better description and matches the binary name.

Reply:
for most people, there's no difference between the "scorer package" and the "language model", and the actual difference is only relevant to a small subset of people who will need more documentation than is on this page.

So, I think the framing of "training" and then "formatting" a language model is most useful. When I've explained this to people using the tools, I use this framing and it works better than anything else in my experience

Reply:
Sorry, I meant using the *verb* to package, ie. "how to train and package a language model for deployment". I think that still captures your intent but matches the tool name, no?

Reply:
ah yes, that's perfect:)